### PR TITLE
config_local: on shutdown, wait for second round of archives

### PR DESCRIPTION
**Pull request by strib**
Noticed this in local testing.

### CODE REVIEW

File: libkbfs/config_local.go [hunk 1]
```diff
@@ -725,16 +725,29 @@ func (c *ConfigLocal) Shutdown() error {
...
 				}
-				if err := fbo.fbm.waitForDeletingBlocks(
-					context.Background()); err != nil {
+				if err := fbo.fbm.waitForDeletingBlocks(ctx); err != nil {
 					return err
 				}
 				if jServer, err := GetJournalServer(config); err == nil {
-					if err := jServer.Wait(context.Background(),
-						fbo.id()); err != nil {
+					if err := jServer.Wait(ctx, fbo.id()); err != nil {
```
Comment (akalin-keybase) [inline libkbfs/config_local.go]: Don't we have a utility function for the `GetJournalServer`/`Wait` combo now?

File: libkbfs/config_local.go [hunk 2]
```diff
@@ -725,16 +725,29 @@ func (c *ConfigLocal) Shutdown() error {
...
+				// The above wait could have resulted in some MD
+				// flushes, so now we have to wait on any archives as
+				// well.
+				if err := fbo.mdFlushes.Wait(ctx); err != nil {
+					return err
+				}
+				if err := fbo.fbm.waitForArchives(ctx); err != nil {
+					return err
+				}
+				if jServer, err := GetJournalServer(config); err == nil {
```
Comment (akalin-keybase) [inline libkbfs/config_local.go]: Why can't this wait trigger another MD flush, and then another archival? Do we need a loop for this?

Comment (strib) [inline libkbfs/config_local.go]: Archives don't produce MDs, so there's no danger of that.  (And this is a test shut down, so we know there won't be any more real writes.)

File: libkbfs/config_local.go (hunk 1 above)
Comment (strib) [inline libkbfs/config_local.go]: Good call, will fix.

**Comment by strib** [comment]
Addressed comments.

**Comment by akalin-keybase** [comment]
lgtm. (I can't seem to find the UI to approve this?)

**Pull request merged**